public/index.html

```diff
@@ -24,12 +22,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap" rel="stylesheet">
```

Comment:
I love Google Fonts 🤩 !

**Small suggestions for the future:**
- Choose two typographies: heading & body text.
- About heading typography, try to find typography with multiple styles (level of bold: `400 - 900`). Because it's a little difficult to distinguish between texts or bold text. Of course, they are only ideas.